plugins/Live/Model.php

```diff
@@ -122,6 +122,33 @@ public function queryLogVisits($idSite, $period, $date, $segment, $offset, $limi
         return $foundVisits;
     }
 
+    public function getMostRecentVisitsDateTime($idSite, $period = null, $date = null): string
```

Comment:
Not a requirement here but it'd be nice if we started adding some docblocks. It always looks so bare and feels unfinished without any method comments.

Reply:
I'll add one, even though I don't consider it much useful to have a doc block if it doesn't provide any additional information...